portal-impl/src/com/liferay/portlet/expando/service/impl/ExpandoValueLocalServiceImpl.java

```diff
@@ -85,10 +83,11 @@ public ExpandoValue addValue(
 			String columnName, long classPK, boolean data)
 		throws PortalException {
 
-		ExpandoTable table = _expandoTableLocalService.getTable(
-			companyId, className, tableName);
+		ExpandoTable table = _expandoTablePersistence.findByC_C_N(
```

Comment:
@slnn Please add an SF commit to rename these variables to follow our convention.

Reply:
Done

Reply:
Hi @dantewang 

I have another concern about the rename for the variable even though the change has been done, do we need to rename these get methods of these localServices? For example, the `getTable()` method return a instance of ExpandoTable, so if we should also remame the method name to `getExpandoTable()`?
```
ExpandoTable table = _expandoTableLocalService.getTable(
			companyId, className, tableName);
```